docs/ibc/overview.md

```diff
@@ -6,170 +6,186 @@ parent:
+specify a timeout height or timeout timestamp, after which a packet can no longer be successfully received on the destination chain.
 
-If the timeout does get reached, then a proof of packet timeout can be submitted to the original chain which can then perform 
-application-specific logic to timeout the packet, perhaps by rolling back the packet send changes (refunding senders any locked funds, etc).
+If the timeout does get reached, then a proof of packet timeout can be submitted to the original chain, which can then perform 
+application-specific logic to timeout the packet, perhaps by rolling back the packet send changes (refunding senders any locked funds, etc.).
 
-In ORDERED channels, a timeout of a single packet in the channel will cause the channel to close. If packet sequence `n` times out, 
-then no packet at sequence `k > n` can be successfully received without violating the contract of ORDERED channels that packets are processed in the order that they are sent. Since ORDERED channels enforce this invariant, a proof that sequence `n` hasn't been received on the destination chain by packet `n`'s specified timeout is sufficient to timeout packet `n` and close the channel.
+- In ORDERED channels, a timeout of a single packet in the channel causes the channel to close. If packet sequence `n` times out, 
+then no packet at sequence `k > n` can be successfully received without violating the contract of ORDERED channels that packets are processed in the order that they are sent. Since ORDERED channels enforce this invariant, a proof that sequence `n` hasn't been received on the destination chain by the specified timeout of packet `n` is sufficient to timeout packet `n` and close the channel.
 
-In the UNORDERED case, packets may be received in any order. Thus, IBC will write a packet receipt for each sequence it has received in the UNORDERED channel. This receipt contains no information, it is simply a marker intended to signify that the UNORDERED channel has received a packet at the specified sequence. To timeout a packet on an UNORDERED channel, one must provide a proof that a packet receipt does not exist for the packet's sequence by the specified timeout. Of course, timing out a packet on an UNORDERED channel will simply trigger the application specific timeout logic for that packet, and will not close the channel.
+- In UNORDERED channels, packets can be received in any order. Thus, IBC writes a packet receipt for each sequence it has received in the UNORDERED channel. This receipt contains no information; it is simply a marker intended to signify that the UNORDERED channel has received a packet at the specified sequence. To timeout a packet on an UNORDERED channel, one must provide a proof that a packet receipt does not exist for the packet's sequence by the specified timeout. Of course, timing out a packet on an UNORDERED channel simply triggers the application-specific timeout logic for that packet, and does not close the channel.
```

Comment:
```suggestion
- In UNORDERED channels, packets can be received in any order. Thus, IBC writes a packet receipt for each sequence it has received in the UNORDERED channel. This receipt contains no information; it is simply a marker intended to signify that the UNORDERED channel has received a packet at the specified sequence. In order to timeout a packet on an UNORDERED channel, a proof is required that a packet receipt **does not exist** for the packet's sequence by the specified timeout deadline. Timing out a packet on an UNORDERED channel prevents any further receiving of the packet, triggers the application-specific timeout logic for that packet, but does not close the channel.
```

Reply:
see how the rewrite is now... let's try to write concisely what is happening. Instead of "deadline" let's use "timeout" (since it's timeout height, not a date-based deadline)

let's say what happens, not what doesn't happen

Very challenging! I feel for our users

Reply:
oh man, now I went and rewrote a bit more for clarity